### DISCUSSION TITLE
In-place scrolling with CTRL+up/down; (CTRL)F2 to add/remove/jump to location

### DISCUSSION THREAD

**Discussion by Shohreh** [thonny/thonny#3105] [General]
Hello,

As a Python newbie (on Windows), I just tried Thonny 4.1.4 and noticed a couple of features I like were missing:
1. Scrolling the text in place using CTRL+up/down arrows
2. Adding/removing locations with CTRL+F2 and jumping to them with F2

Cheers,

**Comment by aivarannamaa** [comment]
Which other editors work like this?

**Comment by Shohreh** [comment]
I obviously didn't keep a list, but UltraEdit and Notepad2 for two.

… and PyScripter :-p

**Comment by aivarannamaa** [comment]
I'll consider adding these features after more pressing issues get solved.
